cli/offchain-commands/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+use sidechain_domain::*;
+use std::error::Error;
+
+#[derive(Clone, Debug, clap::Subcommand)]
+#[allow(clippy::large_enum_variant)]
+pub enum OffchainCmd {
+	/// Get the script addresses for a given sidechain
+	Addresses,
+	/// Register a committee candidate
+	Register,
+	/// Deregister a committee candidate
+	Deregister,
+	/// Insert new D parameter
+	InsertDParameter,
+	/// Update a D parameter
+	UpdateDParameter,
+	/// Set or update permissioned candidates list
+	UpdatePermissionedCandidates,
+	/// Create a new token reserve
+	ReserveCreate,
+	/// Deposit assets to existing reserve
+	ReserveDeposit,
+	/// Empty and remove an existing reserve
+	ReserveHandover,
+	/// Release currently available funds from an existing
+	ReserveRelease,
+}
+
+#[derive(Clone, Debug, clap::Parser)]
+#[command(author, version, about, long_about = None)]
+pub struct SidechainParams {
```

Comment:
We have this already defined the parameters in a chain-params crate and in partner-chains-cli crate (config.rs)